scripts/jira_csv_local_developement.md

```diff
@@ -0,0 +1,55 @@
+# How to run the full process for jira csv locally
+
```

Comment:
It'd be good to have a list of required software here if there's anything special required.